python/cog/server/runner.py

```diff
@@ -283,12 +268,13 @@ def _upload_files(self, output: Any) -> Any:
             raise FileUploadError("Got error trying to upload output files") from error
 
 
-def setup(*, worker: Worker) -> Dict[str, Any]:
+async def setup(*, worker: Worker) -> Dict[str, Any]:
     logs = []
     status = None
     started_at = datetime.now(tz=timezone.utc)
 
     try:
+        # will be async
```

Comment:
Do we need an async yield (`await asyncio.sleep(0)`) in this loop as you've done in the predict loop?

Reply:
I added it in the predict loop because I was getting errors and didn't get any in this loop, but we may as well for completeness